modules/programs/msmtp.nix

```diff
@@ -0,0 +1,96 @@
+{ config, lib, pkgs, ... } @ top:
+
+with lib;
+with builtins;
+
+let
+  cfg = config.programs.msmtp;
+
+  sendMsmtpCommand = account:
+    if cfg.offlineSendMethod == "native" then
+      "${pkgs.msmtp}/bin/msmtpq -C $XDG_CONFIG_HOME/msmtp/config --account=${account.name} -t"
+    else
+      "${pkgs.msmtp}/bin/msmtp $XDG_CONFIG_HOME/msmtp/config --account=${account.name} -t";
```

Comment:
Can't assume that `$XDG_CONFIG_HOME` is set, better use `${config.xdg.configHome}`.